v2.0/releases/index.md

```diff
@@ -6,6 +6,21 @@ The following table shows component versioning for Calico  **{{ page.version }}*
 
 Use the version selector at the top-right of this page to view a different release.
 
+## v2.0.0-rc2
+
+| Component                     | Version                                                                                    |
+|-------------------------------+--------------------------------------------------------------------------------------------|
+| felix                         | [2.0.0-rc4](https://github.com/projectcalico/felix/releases/tag/2.0.0-rc4)                 |
+| calicoctl                     | [v1.0.0-rc2](https://github.com/projectcalico/calico-containers/releases/tag/v1.0.0-rc2)   |
+| calico/node                   | [v1.0.0-rc2](https://github.com/projectcalico/calico-containers/releases/tag/v1.0.0-rc2)   |
+| calico/cni                    | [v1.5.3](https://github.com/projectcalico/calico-cni/releases/tag/v1.5.3)                  |
+| libcalico                     | [v0.19.0](https://github.com/projectcalico/libcalico/releases/tag/v0.19.0)                 |
+| libcalico-go                  | [v1.0.0-rc4](https://github.com/projectcalico/libcalico-go/releases/tag/v1.0.0-rc4)        |
+| calico-bird                   | [v0.2.0-rc1](https://github.com/projectcalico/calico-bird/releases/tag/v0.2.0-rc1)         |
+| libnetwork-plugin             | [v1.0.0-rc3](https://github.com/projectcalico/libnetwork-plugin/releases/tag/v1.0.0-rc3)   |
+| calico/kube-policy-controller | [v0.5.1](https://github.com/projectcalico/k8s-policy/releases/tag/v0.5.1)                  |
+| networking-calico             | [889cfff](http://git.openstack.org/cgit/openstack/networking-calico/tree/?id=889cfff)      |
+
 ## v2.0.0-beta
```

Comment:
Do we want this here still?  I'm easy either way, but the 2.0 docs no longer refer to beta.